test/aspect_test_helper.rb

```diff
@@ -0,0 +1,8 @@
+Spork.prefork do
+  unless ActiveRecord::Base.connection.table_exists?(:test_aspects)
+    ActiveRecord::Base.connection.create_table :test_aspects do |t|
```

Comment:
When using postgres, the models get validated against a db. You can try to mock it out, but it will be just too hard, you will probably need to mock the whole undeluying database adapter instance including all its methods.